docs/contacts/contacts-custom-properties.md

```diff
@@ -0,0 +1,10 @@
+# Custom Properties
+
+Contacts also have the ability to be defined even more specifically with custom properties such as job role or title, these properties allow us not only to have knowledge of our contact, but also allows us to filter that contact should we have specific communication or initiatives that need to be received by a specific department. As the CRM continues to be built out, additional custom properties can be added to reflect the needs of the Cal-ITP team.
```

Comment:
Contacts ~~also~~ have the ability...
Edit: I'd remove 'also' since this is on it's own page its unclear what 'also' is continuing from

`Contacts also have the ability to be defined even more specifically with custom properties`
Edit: Contacts have the ability to be **further refined** using custom properties, such as job role or title. These properties allow us not only to have knowledge of our contact, ****but also allow~~s~~ us to filter by the property if specific reporting is needed**

Reply:
@laneymangan I updated the text with that info. Let me know if there is anything else.